src/modules/explorer/pages/ProposalDetails/index.tsx

```diff
@@ -242,35 +361,55 @@ export const ProposalDetails: React.FC = () => {
                 </Tooltip>
               </Grid> */}
             </Grid>
-            <Grid item>
-              <Grid container justifyContent="space-between" alignItems="center">
-                <Grid item>
-                  {proposal && cycleInfo && (
-                    <Grid container style={{ gap: 20 }}>
-                      <Grid item>
-                        <StatusBadge status={proposal.getStatus(cycleInfo.currentLevel).status} />
-                      </Grid>
-                      <Grid item>
-                        <Typography color="textPrimary" variant="subtitle2">
-                          CREATED BY
-                        </Typography>
-                      </Grid>
-                      <Grid item>
-                        <UserBadge address={proposal.proposer} short={true} />
-                      </Grid>
+            <Grid item container>
+              <Grid
+                container
+                direction={isMobileSmall ? "column" : "row"}
+                justifyContent={isMobileSmall ? "center" : "space-between"}
+                alignItems="center"
+                style={isMobileSmall ? { gap: 18 } : {}}
+              >
+                {proposal && cycleInfo && (
+                  <Grid
+                    item
+                    container
+                    xs={isMobileSmall ? 12 : 9}
+                    justifyContent={isMobileSmall ? "center" : "flex-start"}
+                    spacing={2}
+                    alignItems="flex-end"
+                  >
+                    <Grid item>
+                      {" "}
+                      <StatusBadge status={proposal.getStatus(cycleInfo?.currentLevel).status} />{" "}
```

Comment:
A variable can be created for this so it can be reused - 

```proposal.getStatus(cycleInfo?.currentLevel).status```

Reply:
fixed!